src/components/LastSeen.tsx

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useState } from 'react';
+import api from 'src/service/api';
+import RenderResults from './RenderResults';
+
+const LastSeen = () => {
+    const [data, setData] = useState([]);
+
+    useEffect(() => {
+        api.getDataByIds(api.getHistoryIdList()).then((res: any) => {
+            setData(res);
+        });
+    }, [])
+
+    return (
+        <RenderResults list={data} />
+    );
+};
+
+export default LastSeen;
```

Comment:
pls, add line in the end

Reply:
It is hidden by default, if there is no line then red (-) will be shown